app/metrics.yaml

```diff
@@ -130,3 +130,66 @@ settings_screen:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-08-17"
+
+shortcuts:
+  shortcuts_on_home_number:
+    type: quantity
+    description: |
+      The number of shortcuts the user has on home screen,
+      0, 1, 2, 3 or 4 (maximum)
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5056
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5189
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-20"
+    unit: shortcut(s)
+  shortcut_opened_counter:
```

Comment:
Similar to `shortcuts_on_home_number`, if this metric is relevent "per session" rather than "per day", it would be best to send it in a custom ping. If you are only interested in how many shortcuts are opened per day (or just an average per session), then this is fine as is.

Reply:
Same explanation as above, we only need average usage for all users. Let me know if my arguments are "sane" :)